src/Microsoft.Graph/Requests/Generated/IUserJoinedTeamsCollectionRequest.cs

```diff
@@ -60,7 +60,7 @@ public partial interface IUserJoinedTeamsCollectionRequest : IBaseRequest
         /// </summary>
         /// <param name="expandExpression">The expression from which to calculate the expand value.</param>
         /// <returns>The request object to send.</returns>
-        IUserJoinedTeamsCollectionRequest Expand(Expression<Func<Group, object>> expandExpression);
```

Comment:
And this.

Reply:
https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL1699